### DISCUSSION TITLE
[Feature Request / Bug] Why are the XZ files, not part of the Windows SevenZip project ?

### DISCUSSION THREAD

**Discussion by Smurf-IV** [pocoproject/poco#4186] [Q&A]
And is it possible to have a c++ reference file to take a named file / stream and convert to a named xz / compressed stream please .

i.e. at least the following are missing in the SevenZip 1.12.4 project

![image](https://github.com/pocoproject/poco/assets/2418812/47f19990-9ed1-4acf-b62d-841ddfeb2762)

**Comment by obiltschnig** [comment]
SevenZip is no longer actively maintained.

**Comment by Smurf-IV** [comment]
Is this in the documentation somewhere?
Can it be placed into a Readme in the SevenZip directory structure ?

Something else (Like the equivalent of [Deprecated])?

**Comment by aleks-f** [comment]
@Smurf-IV if it is not released, it is not officially supported
